### ISSUE TITLE
Two versions of graphql-extensions used (0.8.1/0.9.1) by apollo-server-core 2.8.1

### ISSUE THREAD

**Issue by bxt** [apollographql/apollo-server#3116]
# Expected behavior

The latest apollo server packages are compatible with the latest version of `graphql-extensions`.

# The actual behavior

Packages `apollo-cache-control@0.8.1` and `apollo-tracing@0.8.1` are dependent on outdated `graphql-extensions@0.8.1`.

# Reproduction

```
mkdir tmp && cd tmp
npm i graphql
npm i apollo-server-core
npm list graphql-extensions
```

It will give an output like this:

```
`-- apollo-server-core@2.8.1
  +-- apollo-cache-control@0.8.1
  | `-- graphql-extensions@0.8.1  extraneous
  +-- apollo-engine-reporting@1.4.3
  | `-- graphql-extensions@0.9.1  deduped
  +-- apollo-tracing@0.8.1
  | `-- graphql-extensions@0.8.1  extraneous
  `-- graphql-extensions@0.9.1 
```

Notice the "extraneous" entries.

It is a little bit strange to reproduce, since in the repo all packages have `file:` dependencies. It seems to be a problem in the release tooling. As far as I can tell, there were no recent changes to `graphql-extensions` that would even justify a new version.

# The last version of Apollo where the problem did _not_ occur.

It worked with 2.7.1:

```
`-- apollo-server-core@2.7.1
  +-- apollo-cache-control@0.8.1
  | `-- graphql-extensions@0.8.1  deduped
  +-- apollo-engine-reporting@1.4.1
  | `-- graphql-extensions@0.8.1  deduped
  +-- apollo-tracing@0.8.1
  | `-- graphql-extensions@0.8.1  deduped
  `-- graphql-extensions@0.8.1 
```

And stopped working with 2.7.2.

**Comment by kachkaev** [comment]
Same problem after deleting `yarn.lock`, running `yarn install` and starting the dev server. Seems to be caused by `apollo-server-express 2.7.0  → 2.8.1` with all its dependencies.

**Comment by abernix** [comment]
Could you try this again with 2.9.1?

**Comment by abernix** [comment]
Thanks to your reproduction, I can confirm this is fixed by Fixed by 8b21f83286c68fa9a8f63c781244ee511d9a52f5 and f710fe46c90ce22ccc6611050ec21c42e0d833d4 in the latest version of Apollo Server:

```
$ npm ls graphql-extensions
tmp.u0122yJE@1.0.0 /private/var/folders/mq/s5hmqjmx7djftqd_2fqjfn0w0000gn/T/tmp.u0122yJE
└─┬ apollo-server-core@2.9.2
  ├─┬ apollo-cache-control@0.8.3
  │ └── graphql-extensions@0.10.2  deduped
  ├─┬ apollo-engine-reporting@1.4.5
  │ └── graphql-extensions@0.10.2  deduped
  ├─┬ apollo-tracing@0.8.3
  │ └── graphql-extensions@0.10.2  deduped
  └── graphql-extensions@0.10.2
```

Thanks for reporting this originally!

**Comment by bxt** [comment]
@abernix Yes, it works 🎉 thank you so much for fixing this 🙏
